src/02-common-principles.md

```diff
@@ -55,6 +56,39 @@ misunderstanding we clarify them here.
     due to different subject response or randomized nature of the stimuli). Run
     is a synonym of a data acquisition.
 
+## File names consist of key-value pairs (entities)
+
+We define file names with a chain of key-value pairs, each of which we call
+an *entity*. Two prominent examples are the `subject` and `session` entities.
+
+If we want to name a datafile that was collected in a certain `session` from
+a certain `subject`, the name of that datafile will start with the following
+string: `sub-<label>_ses-<label>`.
+
+Note that `sub-<sub-label>` correspondes to the `subject` entity because it has
+the `sub-` "key" and`<sub-label>` "value", where `<sub-label>` would in a real
```

Comment:
I think it should be just `<label>` instead of `<sub-label>` to stay consistent with the rest?

Reply:
I thought so too, but then decided against it to mark the point that the `<label>` is different between`subject` and `session` ... is that not helpful / helpful enough to break the consistency?